### DISCUSSION TITLE
how to make animated character by png file in resource pack?

### DISCUSSION THREAD

**Discussion by BredyAK** [MoreMcmeta/core#114] [Help Wanted]
hi there, how could i use a png file with a `.(more)mcmeta` to display animated characters in a resource pack? what i mean is, i want to edit the file `assets/minecraft/font/default.json` to make some character rendered as image, such as:

```json
{
    "providers": [
        {
            "type": "bitmap",
            "file": "bukera_island:ui/titles/burning_ashes.png",
            "ascent": 16,
            "height": 16,
            "chars": [
                "我"
            ]
        }
    ]
}
```

then if i send "我" in chat, the unicode will become a gif. i've tried to add a `.mcmeta` file or `.moremcmeta` for the source png, but it seems not work, it just display the whole static image.

**Comment by soir20** [comment]
I think the answer here is most likely that it's not supported yet--I haven't explicitly tested with making animated text characters. It is something I'd be willing to look into adding. Could you upload a sample resource pack that shows what you are currently testing, if possible?

**Reply by BredyAK** [reply]
hi there, here is a sample resource pack for Minecraft 1.20.1 that is **wished to work** with MoreMcmeta installed, it only include two features:
- if you send the chinese word "静" in game, you will get a colored zombie head icon.
- if you send the chinese word "动" in game, you are **expected** to get a dynamic furnace icon if MoreMcmeta support animated characters. but it doesn't work now since MoreMcmeta is not supported animated characters currently.

[mm_animated_characters_sample-v2.zip](https://github.com/user-attachments/files/18327174/mm_animated_characters_sample-v2.zip)
